Oneirocom/Magick
# Hotfix for rest nodes

## What Changed:

These are some nice fixes for prod we should probly merge in early if we can.

## How to test:

Make sure avatar plugin and  agentwindow fetch requests are working.
Make sure rest nodes are working.

## Additional information:
N/A
